cropper/src/main/java/com/canhub/cropper/CropException.kt

```diff
@@ -0,0 +1,10 @@
+package com.canhub.cropper
+
+import android.net.Uri
+
+sealed class CropException(message: String) : Exception(message) {
+    class CancellationException : CropException("cropping has been cancelled by the user")
+    class FailedToLoadBitmapException(uri: Uri, message: String?) :
+        CropException("Failed to load sampled bitmap: $uri\r\n$message")
+    class FailedToDecodeImageException(uri: Uri) : CropException("Failed to decode image: $uri")
```

Comment:
Love it! let's just add a prefix to all messages, like `crop: cropping has been cancelled by the user`

Small amount of libraries use this, but helpful to debug!

Reply:
ok it's done